coordinator/src/main/scala/filodb.coordinator/ActorPlanDispatcher.scala

```diff
@@ -44,7 +42,15 @@ case class ActorPlanDispatcher(target: ActorRef, clusterName: String) extends Pl
           emptyPartialResult
         })
       } else {
-        val fut = (target ? plan.execPlan) (t).map {
+        // HACK
```

Comment:
Is this really a hack?

Reply:
yes, this is a hack, not meant to be committed, this PR is just a POC that shows how proto exec plans can works end to end and solicit the feedback. Here, for example the question is how would we want to proceed with the configuration switch that would enable proto serialization for the exec plans. I think the most preferable option is to pass it as a query parameter, smth like protoExec=true.